sync_issues_to_jira/sync_issue.py

```diff
@@ -458,4 +457,21 @@ def _get_jira_comment_body(gh_comment, body=None):
 
 def _get_jira_label(gh_label):
     """ Reformat a github API label item as something suitable for JIRA """
+    # ignore status, resolution and platform: windows labels
+    if _check_issue_label(gh_label["name"]) is None:
+        return
     return gh_label["name"].replace(" ", "-")
+
+def _check_issue_label(label):
+    """
+    Ignore labels that start with "Status:" and "Resolution:". These labels are
+    mirrored from Jira issue and should not be mirrored back as labels
+    """
+    ignore_prefix = ("status:", "resolution:")
+    if label.lower().startswith(ignore_prefix):
+        return None
+    
+    if label.lower() == "platform: windows":
+        return None
+
+    return label
```

Comment:
Nitpick, it's a little odd that this function returns a string or None but result is only ever used as Truthy/Falsy.